### DISCUSSION TITLE
新版播放页下如何处理 `快速收藏` 和 `外置稍后再看` 导致的折行问题

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#4173] [提问 / Q&A] [answered]
不知道是不是因为新版播放页的这个分享组件要给光标悬停后出现的 `点击复制链接` 占位，加上新版播放页的三连按钮的间距好像比旧版大了，单单只开启 `外置稍后再看` 或 `启用快速收藏`，`稿件投诉`、`记笔记` 和更多菜单按钮也都会折到下一行去：
![image](https://user-images.githubusercontent.com/49396421/235420425-6c73092a-3335-42a6-b715-4b21a790e6d9.png)
![image](https://user-images.githubusercontent.com/49396421/235420335-85dd58c4-aecc-406e-9a47-53fe3ea0a364.png)
![image](https://user-images.githubusercontent.com/49396421/235458914-6001693c-984c-4f8f-a768-d523227d9879.png)

不开 `隐藏视频分享` 的情况下，还能不能再在缩一缩避免折行？

还是说我不如等把 `稿件投诉` 和 `记笔记` 收进更多菜单的功能：#4124

**Comment by the1812** [comment]
咋还会折行... 我记得上个版本刚处理过一次

**Reply by the1812** [reply, marked as answer]
我把 #4124 做一下吧, 应该能解决

**Reply by Geeyun-JY3** [reply]
可能是我屏幕太窄了吧😂

而且没记错旧版播放页的分享组件光标悬停后不会出现的 `点击复制链接`，我个人感觉旧版播放页的三连按钮间距也没那么大。

**Reply by Geeyun-JY3** [reply]
前几天破站好像又改了改，改后我这边变成只有字会换行了：
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/ebc10309-b15b-42bd-9b1f-bbaa18f849a8)
